src/sentry/conf/server.py

```diff
@@ -2570,12 +2562,7 @@ def build_cdc_postgres_init_db_volume(settings: Any) -> dict[str, dict[str, str]
     ),
     "clickhouse": lambda settings, options: (
         {
-            "image": "ghcr.io/getsentry/image-mirror-yandex-clickhouse-server:20.3.9.70"
-            if not APPLE_ARM64
-            # altinity provides clickhouse support to other companies
-            # Official support: https://github.com/ClickHouse/ClickHouse/issues/22222
-            # This image is build with this script https://gist.github.com/filimonov/5f9732909ff66d5d0a65b8283382590d
-            else "ghcr.io/getsentry/image-mirror-altinity-clickhouse-server:21.6.1.6734-testing-arm",
+            "image": "ghcr.io/getsentry/image-mirror-altinity-clickhouse-server:21.8.13.1.altinitystable",
```

Comment:
this fails locally on my M1 with `Instruction check fail. The CPU does not support SSSE3 instruction set.`

Reply:
20.3.9.70 doesn't work on colima x86_64 (guessing this is because it's based on alpine by default)

```
dirname: missing operand
Try 'dirname --help' for more information.
```

Reply:
So - we need to run linux/amd64 clickhouse images on colima aarch64 via rosetta.